plugins/python/src/main/kotlin/io/paddle/plugin/python/dependencies/TempVenvManager.kt

```diff
@@ -72,12 +72,13 @@ class TempVenvManager private constructor(val venv: VenvDir, val project: Paddle
     private val interpreterPath: Path
         get() = venv.getInterpreterPath(project)
 
-    fun install(pkg: PyPackage): ExecutionResult {
+    fun install(pkg: PyPackage, disableCache: Boolean = false): ExecutionResult {
```

Comment:
Looks like you could replace it with smth like `data class PipInstallArgs(val noDeps: Boolean = true, val noCacheDir: Boolean = false)`, and add some factory-method to build a list of args for pip command. IMO this should look better than following `listOf(...)` stuff.